CHANGELOG.md

```diff
@@ -2,638 +2,638 @@
+- Ignore routing specs for DescribeClass cop. ([@nijikon])
+- Move rubocop dependency to runtime. ([@nijikon])
+- Update to rubocop 0.39.0. ([@nijikon])
 
 ## 1.4.0 (2016-02-15)
 
-* Update to rubocop 0.37.2. ([@nijikon][])
-* Update ruby versions we test against. ([@nijikon][])
-* Add `RSpec::NotToNot` cop. ([@miguelfteixeira][])
-* Add `RSpec/AnyInstance` cop. ([@mlarraz][])
+- Update to rubocop 0.37.2. ([@nijikon])
+- Update ruby versions we test against. ([@nijikon])
+- Add `RSpec::NotToNot` cop. ([@miguelfteixeira])
+- Add `RSpec/AnyInstance` cop. ([@mlarraz])
 
 ## 1.3.1
 
-* Fix auto correction issue - syntax had changed in RuboCop v0.31. ([@bquorning][])
-* Add RuboCop clone to vendor folder - see #39 for details. ([@bquorning][])
+- Fix auto correction issue - syntax had changed in RuboCop v0.31. ([@bquorning])
+- Add RuboCop clone to vendor folder - see #39 for details. ([@bquorning])
 
 ## 1.3.0
 
-* Ignore non string arguments for FilePathCop - thanks to @deivid-rodriguez. ([@geniou][])
-* Skip DescribeMethod cop for tagged specs. ([@deivid-rodriguez][])
-* Skip DescribeClass cop for feature/request specs. ([@deivid-rodriguez][])
+- Ignore non string arguments for FilePathCop - thanks to @deivid-rodriguez. ([@geniou])
+- Skip DescribeMethod cop for tagged specs. ([@deivid-rodriguez])
+- Skip DescribeClass cop for feature/request specs. ([@deivid-rodriguez])
 
 ## 1.2.2
 
-* Make `RSpec::ExampleWording` case insensitive. ([@geniou][])
+- Make `RSpec::ExampleWording` case insensitive. ([@geniou])
 
 ## 1.2.1
 
-* Add `RSpec::VerifiedDoubles` cop. ([@andyw8][])
+- Add `RSpec::VerifiedDoubles` cop. ([@andyw8])
 
 ## 1.2.0
 
-* Drop support of ruby `1.9.2`. ([@geniou][])
-* Update to RuboCop `~> 0.24`. ([@geniou][])
-* Add `autocorrect` to `RSpec::ExampleWording`. This experimental - use with care and check the changes. ([@geniou][])
-* Fix config loader debug output. ([@geniou][])
-* Rename `FileName` cop to `FilePath` as a workaround - see [#19](https://github.com/nevir/rubocop-rspec/issues/19). ([@geniou][])
+- Drop support of ruby `1.9.2`. ([@geniou])
+- Update to RuboCop `~> 0.24`. ([@geniou])
+- Add `autocorrect` to `RSpec::ExampleWording`. This experimental - use with care and check the changes. ([@geniou])
+- Fix config loader debug output. ([@geniou])
+- Rename `FileName` cop to `FilePath` as a workaround - see [#19](https://github.com/nevir/rubocop-rspec/issues/19). ([@geniou])
 
 ## 1.1.0
 
-* Add `autocorrect` to `RSpec::DescribedClass` cop. ([@geniou][])
+- Add `autocorrect` to `RSpec::DescribedClass` cop. ([@geniou])
 
 ## 1.0.1
 
-* Add `config` folder to gemspec. ([@pstengel][])
+- Add `config` folder to gemspec. ([@pstengel])
 
 ## 1.0.rc3
 
-* Update to RuboCop `>= 0.23`. ([@geniou][])
-* Add configuration option for `CustomTransformation` to `FileName` cop. ([@geniou][])
+- Update to RuboCop `>= 0.23`. ([@geniou])
+- Add configuration option for `CustomTransformation` to `FileName` cop. ([@geniou])
```

Comment:
These changes, from `([@foo][])` to `([@foo])`, are also unfortunate.

Reply:
Hmm, I checked the markdown below and it seems that both are interpreted the same way.

```markdown
- Something. ([@ydah])
* Something. ([@ydah][])

[@ydah]: https://github.com/ydah
```

- Something. ([@ydah])
* Something. ([@ydah][])

[@ydah]: https://github.com/ydah